packages/testing-helpers/src/litFixture.d.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Setups an element synchronously from the provided lit-html template and puts it in the DOM.
+ *
+ * @template {Element} T - Is an element or a node
+ * @param {import('lit-html').TemplateResult} template
+ * @returns {T}
+ */
+export declare function litFixtureSync(template: any): any;
+/**
+ * Setups an element asynchronously from the provided lit-html template and puts it in the DOM.
+ *
+ * @template {Element} T - Is an element or a node
+ * @param {import('lit-html').TemplateResult} template
+ * @returns {Promise<T>}
+ */
+export declare function litFixture(template: any): {};
```

Comment:
@runem do you have an idea why this is `any` and not `TemplateResult`?
probably an issue within ts itself I guess?

Reply:
Adding moduleResolution to the ts compiler options seems to have helped a little for me, I now get the TemplateResult as the return type...

```
const program = createProgram({
  rootNames: fileNames,
  options: {
    module: ModuleKind.ESNext,
    moduleResolution: ModuleResolutionKind.NodeJs,
    target: ScriptTarget.ESNext,
    allowJs: true,
    sourceMap: false,
    declaration: true,
    noEmitOnError: false,
  },
```

Reply:
Upon further investigation TemplateStringArray is also not apparently resolving, and so resulting in an any type for the `html` declaration. I suspect the default libs are not being correctly loaded?

Reply:
@benjamind, @daKmoR I just updated https://github.com/runem/js-to-ts-declaration to overwrite the compiler host in a more gentle way that preserves default libs. This should hopefully solve a couple of problems 🤞